components/popup/test/popup.test.js

```diff
@@ -87,4 +88,31 @@ describe('popup', () => {
 				.equal(false);
 		});
 	});
+
+	describe(`anchor`, function () {
+		it(`should not find anchor element on show`, function () {
```

Comment:
I don't understand from the test what's supposed to happen. 
You set the element in the DOM, you set its `anchor` to "anchor" and then show the popup.
You then expect it to `popup.open` to be false.  
Can the `it` be changed to:
```suggestion
		it(`should not open the popup if anchor does not exist`, function () {
```

Reply:
I cannot see you addressed that. The `it` is not descriptive enough IMO. Can you explain the description?